docs/interfaces/commands-and-shells.rst

```diff
@@ -4,28 +4,25 @@
  Commands and Shells
 #####################
 
-In addition to structured model training workloads handled using :ref:`experiments <experiments>`,
-Determined also supports free-form tasks using *commands* and *shells*. Commands and shells enable
-you to use a Determined cluster and cluster GPUs without writing code that conforms to the trial
-APIs.
+Determined Commands and Shells provide support for running a Determined cluster without writing a
+model. This article describes how to manage GPU-powered batch commands and interactive shells.
 
-Commands execute a user-specified program on the cluster. Commands are useful for running existing
-code in batch mode.
-
-Shells start SSH servers that let you use cluster resources interactively. Shells provide access to
-the cluster in the form of interactive `SSH <https://en.wikipedia.org/wiki/SSH_(Secure_Shell)>`_
-sessions.
+To learn about the Determined Command-Line Interface (CLI) including installation instructions,
+visit the :ref:`cli-ug`.
 
-This document describes the most common CLI and shell commands.
+Commands execute a user-specified program on the cluster. Commands are useful for running existing
+code in batch mode. Shells start SSH servers that let you use cluster resources interactively.
+Shells provide access to the cluster in the form of interactive `SSH
+<https://en.wikipedia.org/wiki/SSH_(Secure_Shell)>`_ sessions.
 
-**********
- Commands
-**********
+*********************
+ Determined Commands
```

Comment:
Given that the title is `Commands and Shells`, I think it's clearer to leave the subheading as `Shells`

Reply:
resolved